evap/staff/forms.py

```diff
@@ -234,27 +255,94 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self._set_responsibles_queryset(self.instance if self.instance.pk else None)
+        if not self.instance.can_be_edited_by_manager:
+            disable_all_fields(self)
 
-        self.fields["responsibles"].queryset = UserProfile.objects.exclude(is_active=False)
-        if self.instance.pk:
-            self.fields["responsibles"].queryset |= UserProfile.objects.filter(
-                pk__in=[user.pk for user in self.instance.responsibles.all()]
+
+class CourseCopyForm(CourseFormMixin, forms.ModelForm):
+    semester = forms.ModelChoiceField(Semester.objects.all())
+    vote_start_datetime = forms.DateTimeField(label=_("Start of evaluations"), localize=True)
+    vote_end_date = forms.DateField(label=_("Last day of evaluations"), localize=True)
+
+    class Meta(CourseForm.Meta):
```

Comment:
If the same meta class is shared between `CourseForm` and `CourseCopyForm`, is there a way to put it in `CourseFormMixin` and inherit it from there?

Reply:
i don't know :)
do you think that's important? (and if yes: do you want to have a look at it?)